lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -177,6 +240,31 @@ def convert_string(value: str) -> str:
     return value
 
 
+def convert_access(value: str) -> str:
+    """
+    Verify that the parameter value is an access scope. Currently this means
```

Comment:
The routine name is `convert_access`, the docstring uses `Verify` which implies no action taken, but the `Returns` section says it returns `the validated access string`.

Perhaps the docstring could read, `Verify that the parameter value is an access scope, returning the converted access string.`?

Reply:
Yes, the description should be better. OK, now I need to decide whether to do that now and prolong this PR, or hold it for the next one. (Sigh.)